renderers/games/teams.py

```diff
@@ -39,17 +18,17 @@ def render_team_banner(
     accent_coords["home"] = layout.coords("teams.accent.home")
 
     for team in ["away", "home"]:
+        color = home_colors['home'] if team == "home" else away_colors['home']
```

Comment:
I know we have to match the existing configs for the time being but this was pretty confusing at first glance why `home` is always the key

Reply:
yeah it’s a wart. It should probably be “primary”, “text”, “highlight” as the names